yarn.lock

```diff
@@ -668,122 +668,160 @@
+  dependencies:
+    "@glimmer/wire-format" "^0.38.5-alpha.6"
+    "@simple-dom/interface" "1.4.0"
+
+"@glimmer/low-level@^0.38.5-alpha.1.6", "@glimmer/low-level@^0.38.5-alpha.6":
+  version "0.38.5-alpha.6"
+  resolved "https://registry.yarnpkg.com/@glimmer/low-level/-/low-level-0.38.5-alpha.6.tgz#189cd21d6c526293b41eaf7f79f58a33369d32a6"
+  integrity sha512-xVy56oEu8ogIrSW2eQJvOaP0sO4kOf11s6WG1CblhM0Sout1u7FbxAtjjyphDEcT+tXvVfB5WK9M+SwgcJAW9Q==
+
+"@glimmer/node@0.38.5-alpha.1.6":
+  version "0.38.5-alpha.1.6"
+  resolved "https://registry.yarnpkg.com/@glimmer/node/-/node-0.38.5-alpha.1.6.tgz#ae656ef99b27db090917f526274c980f26c9e2c3"
+  integrity sha512-ml16x1tF4eL7xSzsGPfXtPUEQ7Uyzth4hk9jxdWoc7wNBJLlChglGij0pCGbXwe839GkJpvJ7l1AYqB0GUya3Q==
+  dependencies:
+    "@glimmer/interfaces" "^0.38.5-alpha.1.6"
+    "@glimmer/runtime" "^0.38.5-alpha.1.6"
+
+"@glimmer/opcode-compiler@0.38.5-alpha.1.6":
+  version "0.38.5-alpha.1.6"
+  resolved "https://registry.yarnpkg.com/@glimmer/opcode-compiler/-/opcode-compiler-0.38.5-alpha.1.6.tgz#35a4c2bd966d711f76f5a9198153626c79cdc4f1"
+  integrity sha512-BZbhfJROiIQrJg68LDPhUAKJ10E9SJ0LiYti++0WdwIpTFBie5ukFWqiC2Lf7OOjBLiz//LCqe5aTAmA0l0vCg==
+  dependencies:
+    "@glimmer/encoder" "^0.38.5-alpha.1.6"
+    "@glimmer/interfaces" "^0.38.5-alpha.1.6"
+    "@glimmer/program" "^0.38.5-alpha.1.6"
+    "@glimmer/reference" "^0.38.5-alpha.1.6"
+    "@glimmer/util" "^0.38.5-alpha.1.6"
+    "@glimmer/vm" "^0.38.5-alpha.1.6"
+    "@glimmer/wire-format" "^0.38.5-alpha.1.6"
+
+"@glimmer/program@0.38.5-alpha.1.6":
+  version "0.38.5-alpha.1.6"
+  resolved "https://registry.yarnpkg.com/@glimmer/program/-/program-0.38.5-alpha.1.6.tgz#2d4d7c868f847f07e47ee3c5860447059a1be70f"
+  integrity sha512-5caiuwYtSG489+IZVU4N5iquZlVyzDaZaMZBSivMal0yuxTMxlOW6oAnz+0J4t73T6Et+BoAVvGHctbIsTJk3A==
+  dependencies:
+    "@glimmer/encoder" "^0.38.5-alpha.1.6"
+    "@glimmer/interfaces" "^0.38.5-alpha.1.6"
+    "@glimmer/util" "^0.38.5-alpha.1.6"
+
+"@glimmer/program@^0.38.5-alpha.1.6", "@glimmer/program@^0.38.5-alpha.6":
+  version "0.38.5-alpha.6"
+  resolved "https://registry.yarnpkg.com/@glimmer/program/-/program-0.38.5-alpha.6.tgz#df0e75acd87789b7c4dd81f72fce6ccb51e05180"
+  integrity sha512-ErDLGS6zpc6zIh4so/xRpK+/9ejV9nqgVVCd7bgt9Qc3hUgStG0mQmz1JUlYPW1wVD155agPNyeA2zwoROTpiA==
+  dependencies:
+    "@glimmer/encoder" "^0.38.5-alpha.6"
+    "@glimmer/interfaces" "^0.38.5-alpha.6"
+    "@glimmer/util" "^0.38.5-alpha.6"
+
+"@glimmer/reference@0.38.5-alpha.1.6":
+  version "0.38.5-alpha.1.6"
+  resolved "https://registry.yarnpkg.com/@glimmer/reference/-/reference-0.38.5-alpha.1.6.tgz#cb4234480226a40ae71485c5cc5c619a28d2118f"
+  integrity sha512-1vagyzKtThtMTiMerpsNMP3qSDrq9vlCB56g7sJyb4cdU4iwVF1F27XR4bzCF3R9Md+PzXXgXPEtH8ohGr/7Xg==
+  dependencies:
+    "@glimmer/util" "^0.38.5-alpha.1.6"
+
+"@glimmer/reference@^0.38.5-alpha.1.6", "@glimmer/reference@^0.38.5-alpha.6":
+  version "0.38.5-alpha.6"
```

Comment:
??????

Reply:
WAT, does yarn not know how to deal with this?!